tests/Resources/ti.ui.textfield.addontest.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2011-Present by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env titanium, mocha */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+var should = require('./utilities/assertions'),
```

Comment:
<!--
  0 failure: 
  1 warning:  tests/Resources/t...
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.ui.textfield.addontest.js;
  Line: 10;
-->

- :warning: tests/Resources/ti.ui.textfield.addontest.js line 10 – 'should' is assigned a value but never used. (no-unused-vars)